share/models/change.py

```diff
@@ -89,7 +93,7 @@ def accept(self, save=True):
         ret = []
         with transaction.atomic():
             self._changes_cache = list(self.changes.all())
-            for c in self._changes_cache:
+            for c in sorted(self._changes_cache, key=lambda c: c.type):
```

Comment:
Ah! stable is the word that I was looking for. Can you just throw a comment in that `sorted` is stable so my hair doesn't stand on end whenever I read that?

I know I read that it is somewhere but it wasn't in the docstring:
```
Help on built-in function sorted in module builtins:

sorted(iterable, key=None, reverse=False)
    Return a new list containing all items from the iterable in ascending order.

    A custom key function can be supplied to customise the sort order, and the
    reverse flag can be set to request the result in descending order.
```

But:
```
Timsort is a hybrid stable sorting algorithm... Timsort has been Python's standard sorting algorithm since version 2.3.
```